actionpack/lib/action_dispatch/http/url.rb

```diff
@@ -354,6 +354,17 @@ def port_string
         standard_port? ? '' : ":#{port}"
       end
 
+      # Returns a number \port suffix like 8080 based on the SERVER_PORT
+      #
+      #   class Request < Rack::Request
+      #     include ActionDispatch::Http::URL
+      #   end
+      #
+      #   req = Request.new 'SERVER_PORT' => '80'
```

Comment:
No, that part should be a hash. It is the same as `actionpack/test/dispatch/request_test.rb#366`